config/Dockerfiles/tests.d/dummy/05-dummy-multi-file-hook

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -xe
+
+# Verify --no-hooks and --ignore-failed-hooks on dummy provider
+# distros.exclude: none
+# providers.include: dummy
+# providers.exclude: none
+
+DISTRO=${1}
+PROVIDER=${2}
+
+PINFILE="PinFile"
+TARGET="dummy_target"
+WORKSPACE_PATH="docs/source/examples/workspaces/dummy-multi-file-hook/"
+
+function clean_up {
+    set +e
+    linchpin -w "${WORKSPACE_PATH}" -p "${PINFILE}" -v destroy "${TARGET}"
+    D_RC=0
+    D_RC=(${?} -o ${D_RC})
+    if [ ${D_RC} -ne 0 ]; then
+        exit ${D_RC}
+    fi
+}
+trap clean_up EXIT SIGHUP SIGINT SIGTERM
+
+if [ -e /tmp/dummy.hosts ]; then
+    rm /tmp/dummy.hosts
+fi
+
+# testing no-hooks flag
+echo "Testing multi file hook"
+linchpin -w "${WORKSPACE_PATH}" -p "${PINFILE}" -v up "${TARGET}"
+
+cat /tmp/ex_hook.txt
```

Comment:
We're getting closer...  Can you actually check the output of the file to verify it contains the data that should have been written out by the second hook?  And if it doesn't exit non-zero.